packages/babel-types/src/definitions/es2015.js

```diff
@@ -568,6 +568,19 @@ defineType("TemplateLiteral", {
       validate: chain(
         assertValueType("array"),
         assertEach(assertNodeType("TemplateElement")),
+        function(node, key, val) {
+          if (
+            Array.isArray(node.expressions) &&
```

Comment:
Nit: If we move this to `expressions` validation instead, we won't have to check that `node.quasis` is an array because that will already have been validated.